charts/ocis/templates/sharing/deployment.yaml

```diff
@@ -78,9 +78,22 @@ spec:
             - name: SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD
               value: {{ .Values.features.sharing.publiclink.writeableShareMustHavePassword | quote }}
 
-            # TODO: does not exist currently. Probably should be there because we also have SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD!?
-            # - name: SHARING_PUBLIC_SHARE_MUST_HAVE_PASSWORD
-            #   value: {{ .Values.features.sharing.publiclink.shareMustHavePassword | quote }}
+            - name: SHARING_PUBLIC_SHARE_MUST_HAVE_PASSWORD
+              value: {{ .Values.features.sharing.publiclink.shareMustHavePassword | quote }}
+
+            # password policies
+            - name: SHARING_PASSWORD_POLICY_MIN_CHARACTERS
+              value: {{ .Values.features.sharing.passwordPolicies.minCharacters | quote }}
+            - name: SHARING_PASSWORD_POLICY_MIN_LOWERCASE_CHARACTERS
+              value: {{ .Values.features.sharing.passwordPolicies.minLowerCharacters | quote }}
+            - name: SHARING_PASSWORD_POLICY_MIN_UPPERCASE_CHARACTERS
+              value: {{ .Values.features.sharing.passwordPolicies.minUpperCharacters | quote }}
+            - name: SHARING_PASSWORD_POLICY_MIN_DIGITS
+              value: {{ .Values.features.sharing.passwordPolicies.minDigits | quote }}
+            - name: SHARING_PASSWORD_POLICY_MIN_SPECIAL_CHARACTERS
+              value: {{ .Values.features.sharing.passwordPolicies.minSpecialCharacters | quote }}
+            - name: SHARING_PASSWORD_POLICY_BANNED_PASSWORDS_LIST
+              value: /etc/ocis/sharing-banned-passwords.txt
```

Comment:
You're not mounting the ConfigMap in the `volumeMounts:` please compare to the frontend service, where the ConfigMap is properly mounted

Reply:
Added in https://github.com/owncloud/ocis-charts/pull/445/commits/f362c97b23581ee7e7e769d3cf90cf679829ce53